test/unit/gateways/adyen_test.rb

```diff
@@ -916,9 +913,8 @@ def test_successful_auth_application_info
     response = stub_comms do
       @gateway.authorize(@amount, @credit_card, options)
     end.check_request do |_endpoint, data, _headers|
-      assert_equal 'Acme', JSON.parse(data)['applicationInfo']['externalPlatform']['name']
-      assert_equal '1', JSON.parse(data)['applicationInfo']['externalPlatform']['version']
-      assert_equal 'abc', JSON.parse(data)['applicationInfo']['externalPlatform']['integrator']
+      assert_equal 'Spreedly', JSON.parse(data)['applicationInfo']['externalPlatform']['name']
```

Comment:
I may be wrong, but IIRC, we shy away from Spreedly-specific values in AM. @curiousepic ?

Reply:
ahhh, oops you're so right. I can change this to a generic name